projects/hslayers/src/components/add-data/file/raster-images/raster-images.component.ts

```diff
@@ -25,7 +26,8 @@ export class HsFileGeotiffComponent
   }
   ngOnInit(): void {
     this.baseDataType = this.dataType;
-    this.acceptedFormats = '.tif, .tiff, .gtiff, .zip';
```

Comment:
`.gtiff` no longer accepted?

Reply:
@jmacura Well I checked layman expected types and gtiff is not included there, so I removed it